### ISSUE TITLE
 CountUp bug and accompanying fix code!

### ISSUE THREAD

**Issue by clayzx** [farhanfadila1717/slide_countdown#61]
When set to countUp mode, there is an issue with carryover resulting in three-digit numbers, for example, 19 would become 110 in the next frame.

Fix code, raw_digit_item.dart file:
```
int minMax(int value) {
    if (widget.countUp) {
      if (value == maxDigit && !currentAndNextIsSame) {
        return 0;
      }

      if (currentAndNextIsSame) {
        return value;
      }

      return value + 1;
    }

    if (value == 0 && !currentAndNextIsSame) {
      return maxDigit;
    }

    if (currentAndNextIsSame) {
      return value;
    }

    return max(value - 1, 0);
  }
```

hope this can be merged into the main branch in the next release.

**Comment by farhanfadila1717** [comment]
thanks @clayzx for filling this issue, sure, i'll added your code next release.

**Comment by farhanfadila1717** [comment]
Already released on pubdev v[1.6.0](https://pub.dev/packages/slide_countdown/versions/1.6.0).
